pkgs/applications/version-management/git-and-tools/git/default.nix

```diff
@@ -18,6 +18,7 @@
 , withLibsecret ? false
 , pkg-config, glib, libsecret
 , gzip # needed at runtime by gitweb.cgi
+, hardcodeSsh ? false
```

Comment:
Yeah, `sshSupport` or `withSsh` does not really fit because you can use ssh remotes just fine when this is false. If someone knows a better name let me know.

Reply:
I would still call it `withSsh`. I think that can be taken as this package isn't alone enough for SSH support. I agree that it is stretching the usual meaning a bit though.

Reply:
Fine for me. Pushing after I have generated the doc and then merging.